### ISSUE TITLE
Escape sequences

### ISSUE THREAD

**Issue by CantThinkOfAUserName** [marcIhm/yabasic#40]
test.yab:
`print "\\"
`
```
yabasic test.yab
---Error in test.yab, line 1:
---Error: String not terminated
   print "\\"
             ^~~~~
---Error: Program not executed
```

I expected to see a single slash but recieved the above error instead.

test.yab:
`print "\\""`
output:
`\"`

**Comment by marcIhm** [comment]
Yep, confirmed. Will fix this.

**Comment by marcIhm** [comment]
Fixed with Version 2.86.6
